### PR TITLE
UIIN-1098 Select and move holdings with items or items to another instance

### PR DESCRIPTION
**Pull request by uladislausamets**
…tance

## Purpose
https://issues.folio.org/browse/UIIN-1098

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/ViewInstance.js [hunk 1]
```diff
@@ -114,6 +115,7 @@ class ViewInstance extends React.Component {
 
     this.state = {
       marcRecord: null,
+      pkuginOpened: false,
```
Comment (AndreiShumski2) [inline src/ViewInstance.js:118]: typo? `pluginOpened`

File: package.json [hunk 2]
```diff
@@ -629,16 +629,17 @@
     "final-form": "^4.18.2",
     "final-form-arrays": "^3.0.1",
     "lodash": "^4.17.4",
+    "moment": "~2.24.0",
     "prop-types": "^15.5.10",
     "query-string": "^5.0.0",
+    "react-beautiful-dnd": "^13.0.0",
```
Comment (NikitaSedyx) [inline package.json:635]: why is it added in your PR?

File: src/ViewInstance.js [hunk 3]
```diff
@@ -173,6 +175,13 @@ class ViewInstance extends React.Component {
     });
   };
 
+  selectInstanse = (instance) => {
+    const { location: { search } } = this.props;
+
+    this.setState({ pluginOpened: false });
```
Comment (NikitaSedyx) [inline src/ViewInstance.js:181]: what plugin?

File: src/ViewInstance.js [hunk 4]
```diff
@@ -256,6 +265,14 @@ class ViewInstance extends React.Component {
     return ref || {};
   };
 
+  openFindInstancePlugin = () => {
```
Comment (NikitaSedyx) [inline src/ViewInstance.js:268]: toggle?

File: src/components/InstancePlugin/InstancePlugin.js [hunk 5]
```diff
@@ -4,21 +4,18 @@ import { FormattedMessage } from 'react-intl';
...
-
-const InstancePlugin = ({ onSelect }) => (
-  <div className={css.marginTop}>
+const InstancePlugin = ({ onSelect, ...props }) => (
+  <>
     <Pluggable
+      {...props}
       aria-haspopup="true"
       dataKey="instances"
-      searchButtonStyle="default"
```
Comment (NikitaSedyx) [inline src/components/InstancePlugin/InstancePlugin.js:14]: why did you remove them? plus you changed layout styles

**Comment by uladislausamets** [comment]
@mkuklis @VictorSoroka1 could you please review this one?

File: test/bigtest/tests/instance-view-page-test.js [hunk 6]
```diff
@@ -111,6 +111,10 @@ describe('InstanceViewPage', () => {
         expect(InstanceViewPage.headerDropdownMenu.hasDuplicateButton).to.be.true;
       });
 
+      it('should render an move items/holdings button', () => {
```
Comment (VictorSoroka1) [inline test/bigtest/tests/instance-view-page-test.js:114]:
```suggestion
      it('should render a move items/holdings button', () => {
```

File: src/ViewInstance.js [hunk 7]
```diff
@@ -173,6 +175,13 @@ class ViewInstance extends React.Component {
     });
   };
 
+  selectInstanse = (instance) => {
+    const { location: { search } } = this.props;
+
+    this.setState({ findInstancePluginOpened: false });
```
Comment (VictorSoroka1) [inline src/ViewInstance.js:181]: Does it make sense to set the state if the next line is going to navigate away from the page? If it is I would call the navigation in the callback which is the second argument to the `setState` as eventually, it is possible that the warning pop up saying that state is going to be set on the unmounted component.

File: src/ViewInstance.js [hunk 8]
```diff
@@ -686,6 +713,7 @@ class ViewInstance extends React.Component {
       <Pane
         data-test-instance-details
         defaultWidth={paneWidth}
+        style={{ flex: 'auto' }}
```
Comment (VictorSoroka1) [inline src/ViewInstance.js:716]: Is the code below doing the same purpose? If so I suggest using it.
```suggestion
        defaultWidth="fill"
```

Comment (uladislausamets) [reply src/ViewInstance.js:716]: Nope, it's not. defaultWidth sets the wrong value to flex property and style rewrite it.

File: src/ViewInstance.js (hunk 7 above)
Comment (uladislausamets) [reply src/ViewInstance.js:181]: It looks like setState is unnecessary here
